### DISCUSSION TITLE
文章标题修复

### DISCUSSION THREAD

**Discussion by codepzj** [Qexo/Qexo#593] [Ideas]
新增文章之后的文章标题的命名规则应该从config.yml读取，比如说:year/:month之类的，而且文章在保存之后，文章所在的大标题无法进行修改（正上方大标题，即_posts中路径），希望修复

**Comment by codepzj** [comment]
就比如说设置了:year/:month 然后我新建文章就2024/12/_______，填标题，然后保存草稿后，应该让路径是可修改的

**Reply by am-abudu** [reply]
路径已经可以修改了
